server/app/models/host_node.rb

```diff
@@ -31,6 +36,7 @@ class HostNode
   field :agent_version, type: String
   field :docker_version, type: String
   field :connected_at, type: Time
+  field :availability, type: String, default: Availability::ACTIVE
```

Comment:
I avoided the enum since it created bit of a hassle in mongoid upgrade and instead went with enum-like class module thingy.